File: Go/Impressão de árvore de asteriscos/arvore.go
package main

import (
	"fmt"
)

func main() {
	var linhas int
	fmt.Print("Digite o número de linhas da árvore: ")
	fmt.Scanln(&linhas)

	for i := 0; i < linhas; i++ {
		for j := 0; j < linhas-i-1; j++ {
			fmt.Print(" ")
		}
		for j := 0; j < i*2+1; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}
}

File: Python/Impressão de árvore de asteriscos/arvore.py
altura = int(input("Digite a altura da árvore: "))

# loop para imprimir cada linha da árvore
for i in range(altura):
    # imprime espaços em branco antes dos asteriscos
    for j in range(altura-i-1):
        print(" ", end="")
    # imprime os asteriscos para a linha atual
    for j in range(2*i+1):
        print("*", end="")
    # imprime uma quebra de linha para a próxima linha da árvore
    print()


